From c0ebc91b15f49b353d02844f264e5d9fdcbb7550 Mon Sep 17 00:00:00 2001
From: Kevin Kofler <kevin.kofler@chello.at>
Date: Sat, 5 Nov 2016 20:34:13 +0100
Subject: [PATCH] [welcome] Fix RequirementsChecker::checkHasInternet().

The NetworkManager property is called "State", not "state". The call
was always failing, and the method was always returning true as a
result.
---
 src/modules/welcome/checker/RequirementsChecker.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/welcome/checker/RequirementsChecker.cpp b/src/modules/welcome/checker/RequirementsChecker.cpp
index a8207a8..f77847c 100644
--- a/src/modules/welcome/checker/RequirementsChecker.cpp
+++ b/src/modules/welcome/checker/RequirementsChecker.cpp
@@ -347,7 +347,7 @@ RequirementsChecker::checkHasInternet()
                            QDBusConnection::systemBus(), 0 );
 
     bool ok = false;
-    int nmState = nmIntf.property( "state" ).toInt( &ok );
+    int nmState = nmIntf.property( "State" ).toInt( &ok );
 
     if ( !ok || !nmIntf.isValid() )
     { 
